dotnet/src/SemanticKernel/Planning/Plan.cs

```diff
@@ -268,8 +268,14 @@ public async Task<Plan> InvokeNextStepAsync(SKContext context, CancellationToken
             // Update Plan Result in State with matching outputs (if any)
             if (this.Outputs.Intersect(step.Outputs).Any())
             {
-                this.State.TryGetValue(DefaultResultKey, out string? currentPlanResult);
-                this.State.Set(DefaultResultKey, string.Join("\n", currentPlanResult?.Trim(), resultValue));
+                if (this.State.TryGetValue(DefaultResultKey, out string? currentPlanResult))
+                {
+                    this.State.Set(DefaultResultKey, $"{currentPlanResult}\n{resultValue}");
+                }
+                else
+                {
+                    this.State.Set(DefaultResultKey, resultValue);
+                }
```

Comment:
nit.

```suggestion
                this.State.TryGetValue(DefaultResultKey, out string? currentPlanResult)
                this.State.Set(DefaultResultKey, (currentPlanResult is null) ? resultValue : $"{currentPlanResult}\n{resultValue}");
```

Reply:
Heh yes I actually had this originally and then felt the urge to leverage the result of TryGetValue. Will leave as-is for now.